cpp/src/stream_compaction/unique_count.cu

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+#include "cudf/utilities/default_stream.hpp"
```

Comment:
```suggestion
#include <cudf/utilities/default_stream.hpp>
```
Same here

Reply:
Did we add a pre-commit hook or something that added these? I don't even recall manually adding them... @bdice does this sound familiar to you?

Reply:
Not familiar. IDEs sometimes add headers automatically?